apps/openmw/mwgui/savegamedialog.cpp

```diff
@@ -168,9 +169,9 @@ namespace MWGui
                         className = "?"; // From an older savegame format that did not support custom classes properly.
                 }
 
-                title << " (Level " << it->getSignature().mPlayerLevel << " " << className << ")";
+                title << " (#{sLevel} " << it->getSignature().mPlayerLevel << " " << className << ")";
```

Comment:
It's possible that ```className``` contains a markup sequence that needs to be escaped (e.g. '#' )

Therefore, use:
```
                title << " (#{sLevel} " << it->getSignature().mPlayerLevel << " " << MyGUI::TextIterator::toTagsString(className) << ")";
```